test/jquery.js

```diff
@@ -2,7 +2,7 @@
 ( function() {
 	/* global loadTests: false */
 
-	var path = window.location.pathname.split( "test" )[ 0 ],
+	var path = window.location.pathname.substring( 0, window.location.pathname.lastIndexOf( "test" ) ),
```

Comment:
Having two long chains of property accessors makes this hard to follow. I think we'd be better off with something like
```js
var pathname = window.location.pathname,
	path = pathname.slice( 0, pathname.lastIndexOf( "test" ) ),
```